config/rbac/clusterworkloadresourcemappings_viewer_role.yaml

```diff
@@ -0,0 +1,16 @@
+# permissions for end users to view bindablekinds.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+  name: clusterworkloadresourcemappings-viewer-role
```

Comment:
Note to reviewers: This name is correct, `kustomize` adds `service-binding-` prefix to the name in the resulting bundle so the reference from the `ClusterRoleBinding` is valid.